Lib 2.0.2 version, vl53l8cx.h, the sequence does not match initialization

Hi,

What nonsense did you do in the new Lib 2.0.2 version? I only get warnings.
Don't you do any quality control?

Arduino IDE v2.3.6
XIAO ESP32S3 / Arduino ESP32 Nano
Espressif Core Package 3.2.0

```
In file included from g:\Arduino IDE v2.3.6\portable\sketchbook\libraries\STM32duino_VL53L8CX\src\vl53l8cx.cpp:40:
\sketchbook\libraries\STM32duino_VL53L8CX\src\vl53l8cx.h: In constructor 'VL53L8CX::VL53L8CX(SPIClass*, int, int, int, uint32_t)':
\sketchbook\libraries\STM32duino_VL53L8CX\src\vl53l8cx.h:278:9: warning: 'VL53L8CX::cs_pin' will be initialized after [-Wreorder]
  278 |     int cs_pin;
      |         ^~~~~~
\sketchbook\libraries\STM32duino_VL53L8CX\src\vl53l8cx.h:276:10: warning:   'int VL53L8CX::lpn_pin' [-Wreorder]
  276 |     int  lpn_pin;
      |          ^~~~~~~
\sketchbook\libraries\STM32duino_VL53L8CX\src\vl53l8cx.cpp:66:1: warning:   when initialized here [-Wreorder]
   66 | VL53L8CX::VL53L8CX(SPIClass *spi, int _cs_pin, int _lpn_pin, int _i2c_rst_pin, uint32_t _spi_speed)  : dev_spi(spi), cs_pin(_cs_pin), lpn_pin(_lpn_pin), i2c_rst_pin(_i2c_rst_pin),  spi_speed(_spi_speed)
      | ^~~~~~~~
```

The sequence does not match initialization. Why did you always change that? It doesn't work like this.
Please change to
line 275 in vl53l8cx.h
```
    /* Configuration */
    int cs_pin;
    int lpn_pin;
    int i2c_rst_pin;
    uint32_t spi_speed;
```

--- a/src/vl53l8cx.h
+++ b/src/vl53l8cx.h
@@ -272,9 +272,9 @@ class VL53L8CX {
     SPIClass *dev_spi;
 
     /* Configuration */
+    int cs_pin;
     int  lpn_pin;
     int i2c_rst_pin;
-    int cs_pin;
     uint32_t spi_speed;
 
     VL53L8CX_Configuration _dev;

